intermittent_tracker/templates/flaky_tests.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="../static/style.css">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Top 100 Flaky Tests</title>
```

Comment:
This title would have to change to be a bit more generic. Perhaps it could be "Flakiest Tests".